### DISCUSSION TITLE
大佬你好，我想问一下贵主题和heo主题，anzhiyu主题等的区别有哪些细分吗，想试一下这个主题，然后对于butterfly主题的迁移性如何

### DISCUSSION THREAD

**Discussion by w1ndys** [valor-x/hexo-theme-solitude#115] [Q&A]
大佬你好，我想问一下贵主题和heo主题，anzhiyu主题等的区别有哪些细分吗，想试一下这个主题，然后对于butterfly主题的迁移性如何
因为anzhiyu/heo都是butterfly魔改的嘛，咱们这个主题能不能迁移过来（比如butterfly特有的标签外挂）

**Comment by everfu** [comment]
> 大佬你好，我想问一下贵主题和heo主题，anzhiyu主题等的区别有哪些细分吗，想试一下这个主题，然后对于butterfly主题的迁移性如何
> 因为anzhiyu/heo都是butterfly魔改的嘛，咱们这个主题能不能迁移过来（比如butterfly特有的标签外挂）

建议尝试一下，和 Heo 样式相似度高，butterfly还是不太一样的，毕竟刚写完的新主题，很多地方不入 butterfly ，希望多些人使用吧，共同建立生态。
